src/particles/plasma/PlasmaParticleContainerInit.cpp

```diff
@@ -249,6 +251,60 @@ InitParticles (const amrex::IntVect& a_num_particles_per_cell,
 
             old_size += num_to_add;
         }
+        if (m_do_symmetrize) {
+
+            const amrex::Real x_mid2 = 0.5*(ParticleGeom(lev).ProbLo(0) + ParticleGeom(lev).ProbHi(0));
+            const amrex::Real y_mid2 = 0.5*(ParticleGeom(lev).ProbLo(1) + ParticleGeom(lev).ProbHi(1));
```

Comment:
```suggestion
            const amrex::Real x_mid2 = (ParticleGeom(lev).ProbLo(0) + ParticleGeom(lev).ProbHi(0));
            const amrex::Real y_mid2 = (ParticleGeom(lev).ProbLo(1) + ParticleGeom(lev).ProbHi(1));
```
I think it was correct without `0.5`: For `x_mid2=lo+hi` and `x=lo`, `x_mirror` would be `x_mid2-x=lo+hi-lo=hi`. For `x=hi` is `x_mirror=lo+hi-hi=lo`. This shouldn’t affect the test since lo+hi=0 typically.

Reply:
Oh yes, you are totally right. I was somehow confused by the same `x_mid2` because I assumed it would be the middle between `lo` and `hi`.